docparse/chunking_strategies.mdx

```diff
@@ -0,0 +1,88 @@
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, chunking_options={})
+```
+
+Here is an example specifying a particular chunking options:
+
+```python
+from aryn_sdk.partition import partition_file
+with open("mydocument.pdf", "rb") as f:
+   data = partition_file(f, 
+      chunking_options={
+         "strategy": "context_rich",
+         "tokenizer": "openai_tokenizer",
+         "tokenizer_options": {
+            "model_name": "text-embedding-3-small"
+         },
+         "merge_across_pages": True,
+         "max_tokens": 512,
+      }
+   )
+```
+
+## Options 
+The options that you can specify in the `dict` include the following: 
+
+ - `strategy`: A string specifying the strategy to use to for chunking. Valid values are `context_rich`, `mixed_multi_column`, and `maximize_within_limit`. The default and recommended chunker is `context_rich` as `{'strategy': 'context_rich'}`.
+        - Behavior of `context_rich` chunker: The goal of this strategy is to add context to evenly-sized chunks. Creates chunks by combining adjacent elements until the chunk reaches the length of the max-token limit specified. Each chunk will contain the copy of the most recently seen section header or title. Section headers or titles that are back to back will be grouped together as one large section header during chunking. 
+
+        - Behavior of `maximize_within_limit` chunker: The goal of the `maximize_within_limit` chunker is to make the chunks as large as possible. Merges elements into the last most recently merged set of elements unless doing so would make its token count exceed `max_tokens`. In that case, it would keep the new element separate and start merging subsequent elements into that one, following the same rule. All elements that are a result of mergers are assigned the type 'Section.' Merges elements on different pages, unless `merge_across_pages` is set to `False`.
+
+         - Behavior of `mixed_multi_column` chunker: `mixed_multi_column` performs well on documents with text spread across multiple columns such as newspaper and journal articles. The mixed_multi_column chunking strategy correctly identifies and orders elements in columns. This strategy also drops elements that are only one token in size and drops elements in the top and bottom margins of the page (anything in the top and bottom 5% is dropped). The mixed_multi_column strategy always breaks across pages.
+- `max_tokens`: An integer specifying the cutoff for splitting chunks that are too large. Default value is 512.
+- `tokenizer`: A string specifying the tokenizer to use when determining how characters in a chunk are grouped. Valid values are `openai_tokenizer`, `character_tokenizer`, and `huggingface_tokenizer`. Defaults to `openai_tokenizer`.
```

Comment:
Suggestion: 'A string specifying the tokenizer to use when converting text into tokens.'